### ISSUE TITLE
Work out tax savings when salary sacrificing the bonus

### ISSUE THREAD

**Issue by wozniakpawel** [wozniakpawel/cooltaxtool#36]
Work out the NI savings (if you sacrifice the bonus), and how much tax you will get back if you make a voluntary contribution. This is currently kind-of supported, but the results are not super clear.

**Comment by jrothlis** [comment]
See my comment on #29, hopefully the same thing again.

**Comment by wozniakpawel** [comment]
yep @jrothlis it is the same thing, but I will keep both (duplicate) issues open until resolved
